app/components/molecules/section-name/section-name.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import {ComponentMeta, ComponentStory} from '@storybook/react';
+
+import {Menu, Cult, Temple, Community} from './section-name.component';
+
+export default {
+  title: 'Molecules/SectionName',
+} as ComponentMeta<
+  typeof Menu | typeof Cult | typeof Temple | typeof Community
+>;
+
+const MenuTemplate: ComponentStory<typeof Menu> = () => <Menu />;
```

Comment:
why do we need 4 different examples? a single component with the combinations is enough

Reply:
I thought it would be like the Social Media Buttons component, as I commented on discord